### DISCUSSION TITLE
`prettier-plugin-tailwindcss` not working

### DISCUSSION THREAD

**Discussion by Mat-moran** [LazyVim/LazyVim#1397] [General]
Hi,

I have installed the pluing to sort classes in the recommended tailwind way and it is not working. I do not know why, do I need special configuration with Lazy Vim?

This is the plugin:
https://github.com/tailwindlabs/prettier-plugin-tailwindcss

Any ideas?

Thanks 🙏

**Comment by dpetka2001** [comment]
There is an [issue](https://github.com/fsouza/prettierd/issues/352) at `prettierd` repo about not being able to discover global plugins in prettier config. You might want to follow up with that or make a new issue because it seems like it hasn't gotten much traction to move forward.
